dashboard/src/pages/packages/index.vue

```diff
@@ -29,7 +35,10 @@ const openPackage = (id: number) => {
           <th scope="col">UUID</th>
           <th scope="col">Started</th>
           <th scope="col">Location</th>
-          <th scope="col">Status</th>
+          <th scope="col">
```

Comment:
We could use `class="text-nowrap"` to avoid the line break in smaller widths. We may want to do it for all table headers too ... 

```css
th {
  white-space: nowrap;
}
```

Reply:
:bulb: thanks!